VMS/Pages/Homepage/RelatedActivities.razor.scss

```diff
@@ -0,0 +1,189 @@
+﻿@import '../../Resources/scss/common/_variables.scss';
+
+$greyicon: #B4B4B4;
+$grey: #F3F3F5;
+$blue-hv: #0B588c;
+$green-hv: #306C14;
+$grey-hv: #646464;
```

Comment:
Can you explain why you don't move those color variables into **_variables.scss**???